integrations/spark/spark-3.5/openhouse-spark-runtime/build.gradle

```diff
@@ -55,7 +55,9 @@ dependencies {
   fatJarPackagedDependencies(project(path: ':integrations:spark:spark-3.1:openhouse-spark-runtime_2.12', configuration: 'shadow')) {
     transitive = false
   }
-  implementation(project(path: ':integrations:spark:spark-3.1:openhouse-spark-runtime_2.12', configuration: 'shadow'))
+  implementation(project(path: ':integrations:spark:spark-3.1:openhouse-spark-runtime_2.12', configuration: 'shadow')) {
+    exclude group: "org.apache.iceberg", module: "iceberg-spark-runtime-3.1_2.12"
```

Comment:
I thought this one is shaded already?

Reply:
it was supposed to be but i think it was missed. both 3.1 and 3.5 were available on classpath before this exclusion.